src/test/kotlin/com/charleskorn/kaml/YamlReadingTest.kt

```diff
@@ -997,7 +987,6 @@ object YamlReadingTest : Spek({
                         assert(result).toBe(
                             listOf(
                                 SimpleWrapper(SimpleNull),
-                                SimpleWrapper(SimpleUnit(Unit)),
```

Comment:
What's the reason for removing this?

Reply:
`Unit` is no primitive kind anymore. So it doesn't make any sense to test it with the primitive kinds together.